oak-parent/pom.xml

```diff
@@ -1316,5 +1297,53 @@
         </pluginManagement>
       </build>
     </profile>
+    <profile>
+      <id>intellij-build-workaround</id>
```

Comment:
I find it a bit nasty that this workaround profile is actually containing the default configuration when it is not running in IntelliJ. Can we rather leave the default in place without any profile, and override only for IntelliJ (with this profile)?

Reply:
I agree with you in principle. The problem is that I don't know how to remove the `_plugin.scr` instruction once it's configured. Do you know of a way to achieve that?

Reply:
Just look at https://maven.apache.org/guides/introduction/introduction-to-profiles.html#profile-order and https://maven.apache.org/pom.html#advanced_configuration_inheritance